packages/app/src/app/pages/common/Modals/ShareModal/index.tsx

```diff
@@ -198,6 +198,13 @@ export const ShareModal: React.FC<Props> = () => {
             >
               {copied ? 'Copied' : 'Copy Embed Code'}
             </Button>
+            <Field label="Embed url">
+              <Input
+                onFocus={select}
+                value={getEmbedUrl(sandbox, mainModule, state)}
+                readOnly
+              />
+            </Field>
             <Field label="Editor url (also works on Medium)">
```

Comment:
Hey @cherniavskii this look good, just one small request. Remove the old Editor url field to avoid confusion between the two links. We want people to use the embed link for all embeds. the editor url is the same as the browser url, doesn't make sense for it to be there.

Reply:
Sure, makes sense!
Should I keep the "also works on Medium" note for embed url?

Reply:
nope, not needed. thanks!